.travis.yml

```diff
@@ -39,11 +39,7 @@ env:
 
 matrix:
   include:
-    # check code formatting
```

Comment:
Why did you moved this line?
This should be the first check of the whole code base, formatting should be always checked first.

Reply:
error